google-http-client-jdo/pom.xml

```diff
@@ -75,27 +75,25 @@
       <artifactId>google-http-client</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.google.http-client</groupId>
-      <artifactId>google-http-client-test</artifactId>
-      <scope>test</scope>
+      <groupId>org.datanucleus</groupId>
+      <artifactId>datanucleus-core</artifactId>
```

Comment:
It looks like previously this code depended against just the jdo API. Is this core just API, or is it implementation?